sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/command_job.py

```diff
@@ -60,6 +60,8 @@
 
 class CommandJob(Job, ParameterizedCommand, JobIOMixin):
     """Command job.
+    
+    Note that for command components, inputs, outputs, and parameters are accessible as environment variables using the prefix AZUREML_PARAMETER_
```

Comment:
Can you add a more detailed example? It's not clear what you mean. Same for Sweep Job.

Is it `AZUREML_PARAMETER_OUTPUT`, `AZUREML_PARAMETER_INPUT`?